debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/j9/PCStack.java

```diff
@@ -1017,7 +1017,7 @@
 		0x00 /* JBgotow = 200  -- pops: 0 pushes: 0*/ ,
 		0x00 /* JBunimplemented = 201  -- pops: 0 pushes: 0*/ ,
 		0x00 /* JBbreakpoint = 202  -- pops: 0 pushes: 0*/ ,
-		0x01 /* JBdefaultvalue = 203 -- pops: 0 pushes: 1*/ ,
+		0x50 /* JBdefaultvalue = 203 -- pops: 0 pushes: 1*/ ,
 		0x80 /* JBwithfield = 204  -- pops: 0 pushes: 0*/ ,
```

Comment:
Withfield seems to overlap with 0x80 `/* JBaload (16r19) <oslot><read>*/` and defaultvalue overlaps with `0x50 /* JBfstore (16r38) <islot><write>*/`. Moreover, there are existing values of `JBdefaultvalue (16rCB)` and `JBwithfield (16rCC)` defaulted to 0x0. Can the hex for the unimplemented bytecodes just be altered to prevent overlap or is there a standard that we follow regarding the association of an instruction with a bytecode?

Reply:
It looks like there are some extra unimplemented bytecodes. This code should match the VM version in pcstack.c

Reply:
I've compared the code for `defaultvalue` and `withfield` in both `pcstack.c` and `PCstack.java` and they are the same actually